src/assets/dnd/overflowing.svg

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
```

Comment:
can you export the files as plain SVG (save as -> ...) such that all the inkscape metadata is removed and maybe even a manual cleanup afterwards